extensions/ext/GLSL_EXT_demote_to_helper_invocation.txt

```diff
@@ -27,7 +27,16 @@ Number
 
```

Comment:
You should update the revision and last modified date above, and the revision history at the bottom of the file.

Reply:
Done.